azkaban-common/src/main/java/azkaban/server/HttpRequestUtils.java

```diff
@@ -97,12 +101,20 @@ public static ExecutionOptions parseFlowOptions(final HttpServletRequest req)
       execOptions.setPipelineLevel(queueLevel);
     }
 
-    String mailCreator = DefaultMailCreator.DEFAULT_MAIL_CREATOR;
     if (hasParam(req, "mailCreator")) {
-      mailCreator = getParam(req, "mailCreator");
+      final String mailCreator = getParam(req, "mailCreator");
       execOptions.setMailCreator(mailCreator);
     }
 
+    final Map<String, String> slaSettings = getParamGroup(req, PARAM_SLA_SETTINGS);
+    // emails param is optional
+    final String emailStr = getParam(req, PARAM_SLA_EMAILS, null);
+    final List<SlaOption> slaOptions = SlaRequestUtils.parseSlaOptions(flowName, emailStr,
```

Comment:
Could you please add license details to the new class `SlaRequestUtils`?

Reply:
Will do, sorry. I thought I had that configured in my IDE but it must've been lost at some point.